implementations/rust/ockam/ockam_command/src/project/ticket.rs

```diff
@@ -50,8 +50,8 @@ pub struct TicketCommand {
     #[arg(short, long = "attribute", value_name = "ATTRIBUTE")]
     attributes: Vec<String>,
 
-    #[arg(long = "ticket-ttl", value_name = "SECS", conflicts_with = "member")]
-    ticket_ttl: Option<u64>,
+    #[arg(long = "expires-in", value_name = "SECS", conflicts_with = "member")]
+    expires_in: Option<u64>,
```

Comment:
I'd recommend to switch the `u64` to a `Duration`, and parse it with the `duration_parse` so we can write the units in the parameter (e.g. `1m` or `20s`).

[Here's an example](https://github.com/build-trust/ockam/blob/2513cc701aacc9ccb32c715b096aa7d093abd1bd/implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs#L64) of another command that is doing this.

Reply:
What should the  the new arg `value_name` be? `Duration`?

Reply:
Yeah, `Option<Duration>` to be precise.

Reply:
Sorry, I meant this line:
https://github.com/build-trust/ockam/blob/1d227c130fe0dc225eac19dd1111d6efc9be77b9/implementations/rust/ockam/ockam_command/src/project/ticket.rs#L54

I assume that `value_name` here should be something other than `SECS` now that we can offer a duration. Perhaps `DURATION` is sufficient?

Reply:
Sorry, I missread the comment yesterday! Exactly, `DURATION` would make most sense.